webpack/stories/docs/plugins.md

```diff
@@ -69,6 +69,11 @@ Once all the above is set up then the script `npm run install` executed from roo
 Also `npm run lint` behaves similarly.
 
 
+### Entry points
+
+The webpack config respects the main entry point defined in `package.json`. On top of that it loads all files matching `./webpack/.*index.js`. That allows plugins to define multiple independent entry points. This can be useful in special use-cases. For example when you need to mix some parts of webpack processed code into pages that use asset pipeline only.
```

Comment:
The `.* is` a bit ambiguous. A bash shell would match `.somethingindex.js` but not `somethingindex.js` where the code would actually load it (and so would a regex). I'd either make it very clear it's a regex (use `(.*)`) or drop the `.`. The latter is easier IMHO.